fdbclient/include/fdbclient/NativeAPI.actor.h

```diff
@@ -360,19 +360,26 @@ class Transaction : NonCopyable {
 	                                                       const KeySelector& end,
 	                                                       const Key& mapper,
 	                                                       GetRangeLimits limits,
-	                                                       int matchIndex = MATCH_INDEX_ALL,
 	                                                       Snapshot = Snapshot::False,
 	                                                       Reverse = Reverse::False);
 
+	[[nodiscard]] Future<MappedRangeResult> getMappedRangeOptionalIndex(const KeySelector& begin,
```

Comment:
Is this needed just because the new methods are in the `Transaction` interface? Apart from that, I'd expect that `NativeAPI.actor.h` shouldn't even need to keep around both the new and old method